src/Microsoft.VisualStudio.AppDesigner/Resources/Designer.resx

```diff
@@ -233,7 +233,7 @@
     <value>Change property: {0}</value>
   </data>
   <data name="ResourceEditorFactory_GUID" xml:space="preserve">
-    <value>ff4d6aca-9352-4a5f-821e-f4d6ebdcab11</value>
+    <value>81828910-b8b3-4d2b-99a3-067027c180c1</value>
     <comment>{Locked}</comment>
```

Comment:
We may wish to revert this, and instead use the same (legacy) GUID for Resource Explorer. In that way, existing code across the tree and extensions will continue to work, just with the new editor.

https://dev.azure.com/devdiv/DevDiv/_search?action=contents&text=ff4d6aca-9352-4a5f-821e-f4d6ebdcab11&type=code

Reply:
Done.